lib/systems/platforms.nix

```diff
@@ -456,10 +456,9 @@ rec {
   ## Other
   ##
 
-  riscv-multiplatform = bits: {
+  riscv-multiplatform = {
     name = "riscv-multiplatform";
     kernelArch = "riscv";
-    bfdEmulation = "elf${bits}lriscv";
```

Comment:
@JohnAZoidberg it's based off of this which has the "l", presumably for little endian.